server/src/test/java/org/opensearch/search/query/QueryPhaseTests.java

```diff
@@ -309,6 +309,11 @@ public void testMinScoreDisablesCountOptimization() throws Exception {
         dir.close();
     }
 
+    //TODO: The executor of our new SiFi resizable search queue is OpenSearchThreadPoolExecutor which
+    // unfortunately does not keep track of queueSize/EWMA execution time. We should ideally Add EWMA enabled threadpool
+    // executor and reenable the below unit test
+    // EWMATrackingEsThreadPoolExecutor from ES 8.0 and re-enable the below unit test.
```

Comment:
I don't think anything should be taken from ES 8.0 as it is not Apache License Version 2.0.
I would definitely rephrase this comment and refrain from writing things that would lead anyone to believe (wrongfully) that the code is "from" anywhere but your own original design.

Reply:
Sorry for the confusion. We do not have intent to port code from ES 8.0. It was an error in comments and I've already cleared it up